apps/api/migrations/preference-centralization/preference-centralization-migration.spec.ts

```diff
@@ -1,201 +1,7 @@
-    expect(userPreference?.preferences).to.deep.equal({
-      all: { enabled: true, readOnly: true },
-      channels: {
-        push: { enabled: true },
-        chat: { enabled: true },
-        email: { enabled: true },
-        in_app: { enabled: true },
-        sms: { enabled: false },
-      },
-    });
-  });
-
-  it('should migrate workflows with preference settings and critical flag false', async function () {
-    await notificationTemplateRepository.create({
-      _creatorId: session.user._id,
-      _environmentId: session.environment._id,
-      _organizationId: session.organization._id,
-      critical: false,
-      preferenceSettings: { email: true, sms: false, push: true, in_app: true, chat: true },
-    });
-
-    await preferenceCentralization();
-
-    const updatedPreferences = await preferenceRepository.find({
-      _environmentId: session.environment._id,
-      _organizationId: session.organization._id,
-    });
-    expect(updatedPreferences.length).to.equal(2);
-    const workflowPreference = updatedPreferences.find(
-      (preference) => preference.type === PreferencesTypeEnum.WORKFLOW_RESOURCE
-    );
-    const userPreference = updatedPreferences.find(
-      (preference) => preference.type === PreferencesTypeEnum.USER_WORKFLOW
-    );
-
-    expect(workflowPreference?.preferences).to.deep.equal(DEFAULT_WORKFLOW_PREFERENCES);
-    expect(userPreference?.preferences).to.deep.equal({
-      all: { enabled: true, readOnly: false },
-      channels: {
-        push: { enabled: true },
-        chat: { enabled: true },
-        email: { enabled: true },
-        in_app: { enabled: true },
-        sms: { enabled: false },
-      },
-    });
-  });
-
-  it('should migrate workflows without preference settings', async function () {
-    await notificationTemplateRepository.create({
-      _creatorId: session.user._id,
-      _environmentId: session.environment._id,
-      _organizationId: session.organization._id,
-    });
-
-    await preferenceCentralization();
-
-    const updatedPreferences = await preferenceRepository.find({
-      _environmentId: session.environment._id,
-      _organizationId: session.organization._id,
-    });
-    expect(updatedPreferences.length).to.equal(2);
-    const workflowPreference = updatedPreferences.find(
-      (preference) => preference.type === PreferencesTypeEnum.WORKFLOW_RESOURCE
-    );
-    const userPreference = updatedPreferences.find(
-      (preference) => preference.type === PreferencesTypeEnum.USER_WORKFLOW
-    );
-
-    expect(workflowPreference?.preferences).to.deep.equal(DEFAULT_WORKFLOW_PREFERENCES);
-    expect(userPreference?.preferences).to.deep.equal(DEFAULT_WORKFLOW_PREFERENCES);
-  });
+  /**
+   * IMPORTANT: This migration depends on SubscriberPreferencesRepository which is now removed.
+   * Please checkout the `v2.1.0` tag and run the migration from there.
+   * @see https://github.com/novuhq/novu/releases/tag/v2.1.0
+   */
```

Comment:
This migration doesn't work without the `SubscriberPreferencesRepository`, so we leave a note to checkout the relevant tag and run the migration from there before proceeding